tests/ssl-opt.sh

```diff
@@ -13873,6 +13873,10 @@ run_test    "TLS 1.2 ClientHello indicating support for deflate compression meth
             -s "dumping .client hello, compression. (2 bytes)"
 
 # Handshake defragmentation testing
+
+# To warrant that the handhake messages are large enough and need to be split
+# into fragments, the tests require certificate authentication, and will
+# use either the default per configuration certificate or the server5.crt (548 Bytes).
```

Comment:
As discussed above, the party that fragments is always OpenSSL and it always uses server5.crt, either from `O_NEXT_SRV` or from the test data.